src/puppetlabs/ezbake/core.clj

```diff
@@ -32,9 +34,12 @@
 (def config-dir "ext/config/")
 (def system-config-dir "ext/system-config/")
 (def shared-cli-apps-prefix "ext/cli/")
+(def shared-cli-defaults-prefix "ext/cli_defaults/")
 (def docs-prefix "ext/docs/")
 (def build-scripts-prefix "ext/build-scripts/")
 (def terminus-prefix "puppet/")
+(def additional-uberjar-checkouts-dir "target/uberjars")
+(def cli-defaults-filename "ext/cli_defaults/cli-defaults.sh.erb")
```

Comment:
Very small nit - might be good to concatenate the value for `shared-cli-defaults-prefix` into the construction of this variable so that the "ext/cli_defaults/" part only appears once.